server/routes/pubDocument.tsx

```diff
@@ -121,12 +122,17 @@ const getEnrichedPubData = async (options: EnrichedPubOptions) => {
 		};
 	};
 
-	const [docInfo, edges, subscription] = await Promise.all([
+	const currentCollectionPub = pubData.collectionPubs.find((collectionPub) =>
```

Comment:
this works, but feels slightly sloppy. also did not write any tests or anything to confirm this matches what the "normal" api method finds, just manually tested it.

For reference, here is how it is normally found (on the frontend)
https://github.com/pubpub/pubpub/blob/5041cb89f9b9c8b99402887be5f1ed40be203e87/client/utils/collections.ts#L34-L43